src/client/src/tscSubquery.c

```diff
@@ -101,13 +101,12 @@ static bool allSubqueryDone(SSqlObj *pParentSql) {
 
 bool subAndCheckDone(SSqlObj *pSql, SSqlObj *pParentSql, int idx) {
   SSubqueryState *subState = &pParentSql->subState;
-  assert(idx < subState->numOfSub);
```

Comment:
The index may be equal to or larger than subState->numOfSub, due to subqueries (pSubs) being destroyed from another thread concurrently.